CONTRIBUTING.md

```diff
@@ -1,8 +1,11 @@
 # CONTRIBUTING
 
-The libuv project welcomes new contributors. This document will guide you
+The "Port of libuv to IBM® z Systems" project welcomes new contributors. This document will guide you
```

Comment:
No problem with CONTRIBUTING.md being more general but please leave CLA.md as it is, i.e. project specific.